src/transforms/transform-pull-request.ts

```diff
@@ -87,44 +88,60 @@ export const transformPullRequest = async (
 	log?: Logger
 ) =>
 {
-	const { head } = pullRequest;
-
 	const issueKeys = extractIssueKeysFromPr(pullRequest);
+	const {
+		id,
+		user,
+		comments,
+		base,
+		number,
+		updated_at,
+		head,
+		state,
+		merged_at,
+		title,
+		html_url
+	} = pullRequest;
```

Comment:
We use destructuring in most of our codebase and was agreed upon in our former coding conventions sessions: "Preference for destructuring as it’s clearer and part of ES6. Should try and use the standards set by ES6, as most of our app is using it." That said we agreed not to use if object was nested so only pulled keys from the first layer. Overall though, I'd say it improves the payload readability.